extensions/ql-vscode/src/helpers.ts

```diff
@@ -118,7 +118,7 @@ export async function showBinaryChoiceDialog(message: string): Promise<boolean>
   const yesItem = { title: 'Yes', isCloseAffordance: false };
   const noItem = { title: 'No', isCloseAffordance: true };
   const chosenItem = await Window.showInformationMessage(message, { modal: true }, yesItem, noItem);
-  return chosenItem === yesItem;
+  return chosenItem?.title === 'Yes';
```

Comment:
I suppose I agree with comparing string equality feeling safer than reference equality --- but maybe 
```suggestion
  return chosenItem?.title === yesItem.title;
```
?